src/main/java/br/com/digituz/mailer/service/EmailService.java

```diff
@@ -19,13 +28,59 @@
 	@Autowired
 	private EmailRepository emailRepository;
 
+	@Autowired
+	private JavaMailSender javaMailSender;
+
+	@Value("${spring.mail.username}")
+	private String sender;
+
+	@Value("${mailer.replyTo}")
+	private String replyTo;
+
 	private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
 	public void saveEmails(Email email) {
 		this.emailRepository.save(email);
 		logger.info("Email saved");
 	}
 
+	public void sendEmails() throws MessagingException {
+
+		MimeMessage mimeMessage = javaMailSender.createMimeMessage();
+
+		MimeMessageHelper helper = new MimeMessageHelper(mimeMessage, true);
+
+		emailsAll()
+		.stream()
+		.filter(mail -> EmailStatus.NEW.equals(mail.getEmailStatus()))
+		.forEach(mail -> {
+			String[] recipients = mail.getRecipients()
+					.stream()
+					.map(String::new)
+					.toArray(String[]::new);
+			try {
+				helper.setFrom(sender);
+
+				helper.setTo(recipients);
+
+				helper.setSubject(mail.getTitle());
+				helper.setText(mail.getMessage());
+				helper.setReplyTo(replyTo);
+
+				for (Attachment attachment : mail.getAttachments()) {
+					helper.addAttachment(attachment.getFilename(),
+							new ByteArrayResource(attachment.getData()));
+				}
+
+			} catch (MessagingException e) {
+				logger.error(e.getMessage(), e);
+			}
+		});
+
+		javaMailSender.send(mimeMessage);
```

Comment:
Outra, ao invés de deixar o MessagingException "escapar", faz um catch nele. Quando ele for pego, registra como +1 nos attempts e salva o email como não enviado. Depois disso tem que limitar a quantidade de tentativas.